packages/superset-ui-connection/src/callApi/callApi.ts

```diff
@@ -1,5 +1,9 @@
+/* eslint compat/compat: 'off' */
 import 'whatwg-fetch';
 import { CallApi } from '../types';
+import { CACHE_KEY, NOT_MODIFIED, OK } from '../constants';
```

Comment:
sorry only nit with the status constants is that there's no context if you read them in isolation e.g., `OK` = okay what? could we prefix with `STATUS_*` or `HTTP_*` ?

Reply:
Fixed this.